package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
+    "commonmark": "0.27.0",
+    "commonmark-react-renderer": "hmhealey/commonmark-react-renderer#55da1c569c497db7b58d2f4d8c5e6c2fbbf3f2ff",
```

Comment:
alright we can do this at any time, actually I have to do the same with `react-native-search-bar`